cmd/runtimes/mlx/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM mpioperator/base:v0.6.0 AS mpi
+FROM debian:trixie
```

Comment:
We discussed with @tenzen-y that we will use debian image for MLX Runtime for now since we see problems with running OpenMPI in Fedora.
We will investigate later if we require to improve this runtime.

Reply:
This interesting initiative, great job everyone!

But I wonder how you plan to run this on Linux since at the moment MLX only runs on Apple silicon.

Reply:
Thanks @Blaizzy! As you can see this image builds `mlx` and `mlx-data` from source, also MLX already runs some of their CI tests on Linux machines: https://github.com/ml-explore/mlx/blob/main/.circleci/config.yml#L72

I believe, that might be useful for folks who want to experiment with distributed MLX and MPI inside Kubernetes environment.